tests/test_cache.py

```diff
@@ -132,3 +130,85 @@ def test_model_archive_path(self):
                 ),
                 archive_path,
             )
+
+    def _download_test_model_to_cache(self):
+        cache_path = get_cached_path(TEST_MODEL_HANDLE)
```

Comment:
You are missing the `with create_test_cache():` block (see the other tests above).

This function creates a temporary directory to read/write cache files and set the proper environment variable to ensure the cache uses that temporary directory.

Reply:
Thanks for the pointer!! I think I misunderstood along the way. While testing I thought I might not need that because I call this helper from within a `with create_tech_cache():` block

Reply:
My bad. Disregard my comment. I didn't realize this was a helper function and not a test. 

If this is always called within the `create_test_cache` block, then you are good.